.github/workflows/dev-infra.yml

```diff
@@ -1,12 +1,19 @@
 name: DevInfra
 
+# Declare default permissions as read only.
+permissions: read-all
+
 on:
   pull_request_target:
     types: [opened, synchronize, reopened]
 
 jobs:
   labels:
     runs-on: ubuntu-latest
+    permissions:
```

Comment:
We would need to verify, but I think we don't actually need to set permissions here because we don't use the `GithubAction` account token.

Reply:
That appears to be correct.  Permissions have been updated to reflect this.